### PR TITLE
Misc/holo world add scfe pelvis

### PR DESCRIPTION
**Pull request by boonwj**
Added left-scfe-pelvis hologram information into holo world. The hologram was provided by Ka Wai in response to #51.

This PR also fixes some of the legacy paths used in the README file.

### CODE REVIEW

File: Misc/holo_world/README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ It consists of fhir data for the EHR FHIR server, Storage FHIR server and some d
 - Configure `config.cfg`
     - Additional details on configuration can be found in the tool's README.md
 - Insert the demo data
-    - `python az_blob_tool.py - upload-folder holograms ./../holorepo_demo_data/storage_glb`
+    - `python az_blob_tool.py - upload-folder holograms ../../holo_world/storage_glb`
```
Comment (nielsboecker) [inline Misc/holo_world/README.md:33]: First line may as well be something like `# Holo World: How to set up a complete HoloRepository demo infrastructure` or  anything that includes the "Holo World" title

File: Misc/holo_world/storage_data/doc-ref-h105.holostorage.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        "lastUpdated": "2019-08-01T22:42:13.477+00:00",
+        "versionId": "7"
+    },
+    "resourceType": "DocumentReference",
+    "status": "current",
+    "subject": {
+        "reference": "Patient/p102"
+    },
+    "type": {
+        "text": "UPLOAD_EXISTING_MODEL"
```
Comment (nielsboecker) [inline Misc/holo_world/storage_data/doc-ref-h105.holostorage.json]: How about `"GENERATE_FROM_IMAGING_STUDY"`

**Review by nielsboecker — APPROVED** [review]
👍

File: Misc/holo_world/storage_data/doc-ref-h105.holostorage.json (hunk 2 above)
Comment (boonwj) [reply Misc/holo_world/storage_data/doc-ref-h105.holostorage.json]: Changed

File: Misc/holo_world/README.md (hunk 1 above)
Comment (boonwj) [reply Misc/holo_world/README.md:33]: Changed

Comment (nielsboecker) [reply Misc/holo_world/README.md:33]: I know, I alrady merged it 2 hours ago 💃

**Pull request merged**